src/Symfony/Component/Serializer/Normalizer/ConstraintViolationListNormalizer.php

```diff
@@ -34,19 +34,29 @@ public function normalize($object, $format = null, array $context = array())
         foreach ($object as $violation) {
             $violations[] = array(
                 'propertyPath' => $violation->getPropertyPath(),
```

Comment:
A `$propertyPath` variable is declared just after, this could probably be used here after moving the declaration up.

Reply:
good catch, done